islandora_paged_content.module

```diff
@@ -238,3 +238,14 @@ function swaptable_value($element, $input, array $form_state) {
   }
   return array();
 }
+
+/**
+ * Implements hook_islandora_object_ingested().
+ */
+function islandora_paged_content_islandora_object_ingested(AbstractObject $object) {
+  // Only do this check on paged content that have PDFs.
+  if (array_intersect_key(array('islandora:bookCModel', 'islandora:newspaperIssueCModel'), $object->models) && isset($object['PDF'])) {
```

Comment:
I know we are doing something like this already, but we really shouldn't.  We should create a registry hook for page content models. Also maybe manuscripts should be included as in https://github.com/Islandora/islandora_paged_content/blob/0db9de2906219186d1922b065b1be165dafed2f2/includes/utilities.inc#L298 but it shouldn't be as manuscript is a DGI module which just reinforces the hook notion.